Update Shareable Viz docs with sections on creating access key and billing. (#1631)
kedro-org/kedro-viz: docs/source/share_kedro_viz.md

You can host your Kedro-Viz project on Amazon S3. You must first create an S3 bucket and then enable static website hosting. To do so, follow the [AWS tutorial](https://docs.aws.amazon.com/AmazonS3/latest/userguide/HostingWebsiteOnS3Setup.html) to configure a static website on Amazon S3.

> Once the s3 bucket is created, you'll need to creat an (Identity and Access Management) IAM user account, user group, and access keys:

Reviewer: ```suggestion
Once the S3 bucket is created, you'll need to create an Identity and Access Management (IAM) user account, user group, and generate the corresponding access keys. To do so:
```

> Sign in to the [AWS Management Console](https://console.aws.amazon.com/s3/) and create an  IAM user account.

Reviewer: ```suggestion
Sign in to the [AWS Management Console](https://console.aws.amazon.com/s3/) and create an IAM user account.
```

For more information, see the official AWS documentation about [IAM Identities](https://docs.aws.amazon.com/IAM/latest/UserGuide/id.html).

![](../meta/images/kedro_viz_share_credentials1.png)

> Create a user group from the IAM dashboard (ensuring the user group has full access to AWS S3 policy).

Reviewer: ```suggestion
Create a user group from the IAM dashboard, ensuring the user group has full access to the AWS S3 policy.
```

![](../meta/images/kedro_viz_share_credentials2.png)

> "A user group is a collection of IAM users. User groups simplify permissions management because you can grant, change, and remove permissions for multiple users at the same time". For more information, see the official AWS documentation about [IAM user groups](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups.html?icmpid=docs_iam_help_panel).

Reviewer: I don't think we need the copy/paste parts of the AWS documentation into ours.

```suggestion
For more information, see the official AWS documentation about [IAM user groups](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups.html?icmpid=docs_iam_help_panel).
```

Author: Agreed.

Add the IAM user to the user group (this is only possible if the group has been created).

![](../meta/images/kedro_viz_share_credentials3.png)

> Select the user, and select ‘create access key’. Follow the steps and create your access credential.

Reviewer: ```suggestion
Select the user, then select `Create access key`. Follow the steps and create your keys.
```

All permissions and access control are controlled by AWS. It's up to you, the user, if you want to allow anyone to see your project or limit access to certain IP addresses, users, or groups.

You can control who can view your visualisation using [bucket and user policies](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-iam-policies.html) or [access control lists](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acls.html). See the official AWS documentation for more information.

## Billing

> You pay for storing objects in your S3 buckets. This depends on your objects’ size, how long you stored the object during the month, and the storage class.

Reviewer: ```suggestion
You pay for storing objects in your S3 buckets. The amount you pay depends on your objects’ size, how long you stored the object during the month, and the storage class.
```

> See the official  [AWS documentation](https://aws.amazon.com/s3/pricing/?nc=sn&loc=4) for more information. 

Reviewer: ```suggestion
See the official [AWS documentation](https://aws.amazon.com/s3/pricing/?nc=sn&loc=4) for more information. 
```
